docs/resources/task.md

```diff
@@ -0,0 +1,235 @@
+
+-> **Note** `{accelerator}` will be transformed into a node selector requesting `accelerator={accelerator}` and `{count}` will be configured as the **limits** count for `kubernetes.io/gpu`.
+
+## Machine Images
+
+### Generic
+
+This provider offers some common machine images that are roughly the same for
+all the supported cloud providers.
+
+- `ubuntu` - Official Ubuntu LTS image, currently 20.04.
+
+### Cloud-specific
+
+In addition to generic images, it's possible to specify any machine image
+supported by the underlying cloud provider.
+
+#### Amazon Web Services
+
+`{user}@{architecture}:{owner}:{name}`
+
+Fields:
+
+- `{user}` - User name; e.g. `ubuntu`.
+- `{architecture}` — Image architecture; e.g. `x86_64` or `*` for any.
+- `{owner}` - Account number of the image owner; e.g. `099720109477` or `*` for any.
+- `{name}` - Name of the image; e.g. `*ubuntu/images/hvm-ssd/ubuntu-focal-20.04*`.
+
+See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html for more information.
+
+#### Google Cloud Platform
+
+`{user}@{project}/{family}`
+
+Fields:
+
+- `{user}` - User name; e.g. `ubuntu`.
+- `{project}` - Project name; e.g. `ubuntu-os-cloud`.
+- `{family}` — Image architecture; e.g. `ubuntu-2004-lts`.
+
+See https://cloud.google.com/compute/docs/images/os-details for more information.
+
+#### Microsoft Azure
+
+`{user}@{publisher}:{offer}:{sku}:{version}`
+
+Fields:
+
+- `{user}` - User name; e.g. `ubuntu`.
+- `{publisher}` - Image publisher; e.g. `Canonical`.
+- `{offer}` - Image offer; e.g. `UbuntuServer`.
+- `{sku}` - Image SKU; e.g. `18.04-LTS`.
+- `{version}` - Image version; e.g. `latest`.
+
+See https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage for more information.
+
+### Kubernetes
+
+- `{image}` - Any [container image](https://kubernetes.io/docs/concepts/containers/images/#image-names).
+
+## Cloud Regions
+
+### Generic
+
+This provider offers some common cloud regions that are roughly the same for all
+the supported cloud providers.
+
+- `us-east` - United States of America, East.
+- `us-west` - United States of America, West.
+- `eu-north` - Europe, North.
+- `eu-west` - Europe, West.
+
+### Cloud-specific
+
+In addition to generic sizes, it's possible to specify any cloud region
+supported by the underlying cloud provider.
+
+#### Amazon Web Services
+
+- `{region}` - Any cloud-specific region, like `us-east-1`.
+
+See https://aws.amazon.com/about-aws/global-infrastructure/regions_az for more information.
+
+#### Google Cloud Platform
+
+- `{zone}` - Any cloud-specific **zone** — not region — like `us-east1-a`.
+
+See https://cloud.google.com/compute/docs/regions-zones for more information.
+
+#### Microsoft Azure
+
+- `{region}` - Any cloud-specific region, like `eastus`.
+
+See https://azure.microsoft.com/en-us/global-infrastructure/geographies for more information.
+
+### Kubernetes
+
+The `region` attribute will be ignored.
+
+## Quirks
```

Comment:
```suggestion
## Known Issues
```

Reply:
Kernel device trees use the word “quirks” for this kind of outliers and everybody is happy. 🤷🏼‍♂️ Still, “known issues” is clearer.